### ISSUE TITLE
Message body of failed message is lost when messages contain Ids that have characters that must be URL encoded

### ISSUE THREAD

**Issue by andreasohlund** [Particular/ServiceControl#1596]
To repro:

1. Have a MSMQ endpoint process some raw xml `<root></root>`
2. Processing will fail and message moved to the error queue
3. SC will ingest message but body will not display in SP
4. Request a retry, message will go back to endpoint queue but body is now missing

### SC logs during retry

```
2019-02-28 15:42:18.6023|15|Info|ServiceControl.Recoverability.ReturnToSenderDequeuer|Forwarder for batch RetryBatches/47510a4e-fd6a-4553-883e-81669ed615be started receiving messages from Particular.ServiceControl.staging@CARBONX1.
2019-02-28 15:42:21.6943|73|Warn|ServiceControl.Recoverability.ReturnToSender|72802d3f-fa83-4088-8d02-8489a341c6e2\16410: Message Body not found for attempt Id 72802d3f-fa83-4088-8d02-8489a341c6e2\16389

```

My hunch is that since the message comes in as native message the message id will be `{guid}\counter` and that might trip SC up when storing the body.

Not sure yet if body is not stored or if it's just the query that fails to find it

**Comment by andreasohlund** [comment]
Since the body is lost the endpoint will just silently do a no-op on the message without logging.

**Comment by boblangley** [comment]
Fixed by #1598
